packages/mobile/src/transactions/saga.ts

```diff
@@ -48,15 +52,15 @@ const RECENT_TX_RECIPIENT_CACHE_LIMIT = 10
 
 // Remove standby txs from redux state when the real ones show up in the feed
 function* cleanupStandbyTransactions({ transactions }: NewTransactionsInFeedAction) {
```

Comment:
I think we also need to clean up the non-legacy transactions when the actual tx is found in the feed, no?

Reply:
This is done in the next PR: https://github.com/valora-inc/wallet/pull/1669